### DISCUSSION TITLE
Caddy static files e HA

### DISCUSSION THREAD

**Discussion by angelopampalone** [indomus/forum#2343] [GENERALE - Discussioni generiche]
Buon giorno a tutti, spero che qualcuno possa aiutarmi. Vorrei rendere accessibili alcuni files parcheggiati su un raspberry pi alla mia intranet ma non dall'esterno.
Il setup attuale è un raspberry con HA, Caddy (e altro) su docker composse.

Ho pensato di risolvere configurado un file server in Caddy e volevo procedere il due steps successivi:
- prima definire un site che mostri una stringa statica ed una volta ok 
- definire un file_server per il site che pubblichi i files di /var/www/pippo__pluto nella intranet interna.

Ho provato a fare la seguente configurazione in caddy ma dal restart di Caddy HomeAssistant diventa inaccessibile dall'esterno e la response non funziona. 
Ho provveduto a commentare le modifiche ma spero che qualcuno mi possa aiutare visto che la documentazione di Caddy mi è ostica.

```
(https_header) {
  header {
    Strict-Transport-Security "max-age=31536000; includeSubdomains"
    X-XSS-Protection "1; mode=block"
    X-Content-Type-Options "nosniff"
    X-Frame-Options "SAMEORIGIN"
    Referrer-Policy "same-origin"
  }
}

https://xyz.duckdns.org { 
  respond "Saluti dal Crew inDomus.it"
  file_server
}

https://xyz.duckdns.org:9000 { 
  import https_header
  reverse_proxy http://192.168.1.6:8123
}

# *** NO!!! Rende HA inaccessibile dall'esternoCause ***
##Provo a definire un secondo site con una respond; quando ok provo a def gli static files
#localhost:9015 {
#  import https_header
#  respond "Saluti dal Crew inDomus.it"
#}
```

**Comment by snakuzzo** [comment]
https://caddyserver.com/docs/quick-starts/static-files
